pkg/ovs/openflow/ofctrl_bridge_test.go

```diff
@@ -117,3 +118,17 @@ func TestDeleteGroup(t *testing.T) {
 		})
 	}
 }
+
+func TestConcurrentCreateGroups(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+	// Race detector on Windows has limit of 8192 simultaneously alive goroutines.
+	b := NewOFBridge("test-br", GetMgmtAddress(ovsconfig.DefaultOVSRunDir, "test-br"))
+	b.SwitchConnected(newFakeOFSwitch(b))
+	concurrentNum := 8000
+	for i := 0; i < concurrentNum; i++ {
+		go func(index int) {
+			b.CreateGroup(GroupIDType(index))
```

Comment:
You still need waitgroup to ensure all backend goroutines are completed before the test function is Finished.